src/components/TodoList/TodoCreator/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './index.css';
+import * as todoActions from "../../../state/todos/action";
+import {connect} from 'react-redux';
+
+const TodoCreator = ({color, text, fulfilled, todoIndex, setFulfilledState}) => {
+
+    return (
+        <div style={{background: color}} className={'todoCreator-container'}>
+            {fulfilled === true ?
+                <hr className={'todoCreator-line'} color={'#000'}/>
```

Comment:
use  `text-decoration: line-through`

Reply:
i guess we should keep my realization without changes!